docs/resources/docker.md.erb

```diff
@@ -145,20 +165,55 @@ For a full list of available matchers, please visit our [matchers page](https://
       its('Client.Version') { should cmp >= '1.12'}
     end
 
+<br>
 
-### info
+## Properties
 
-`info` returns the parsed result of [docker info](https://docs.docker.com/engine/reference/commandline/info/)
+* `id`, `image`, `repo`, `tag`, `ports`, `command`
 
-    describe docker.info do
-      its('Configuration.Path') { should eq 'value' }
+<br>
+
+## Property Examples
+
+### id
+
+    describe docker_container(name: 'an-echo-server') do
+      its('id') { should_not eq '' }
     end
 
+### image
 
-### object('id')
+  describe docker_container(name: 'an-echo-server') do
+    its('image') { should eq 'busybox:latest' }
+  end
 
-`object` returns low-level information about docker objects. It is calling [docker inspect](https://docs.docker.com/engine/reference/commandline/info/) under the hood.
+### repo
 
-    describe docker.object(id) do
-      its('Configuration.Path') { should eq 'value' }
+    describe docker_container(name: 'an-echo-server') do
+      its('repo') { should eq 'busybox' }
+    end
+
+### tag
+
+    describe docker_container(name: 'an-echo-server') do
+      its('tag') { should eq 'latest' }
     end
+
+### ports
+
+    describe docker_container(name: 'an-echo-server') do
+      its('ports') { should eq "0.0.0.0:1234->1234/tcp" }
```

Comment:
Is that really the output of `ports`?...yikes...

Reply:
Yesssss.....